[2024-03-02T00:05:28.926+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: stock_analyzer.insert_irctc_data_into_db manual__2024-03-02T00:05:22.461486+00:00 [queued]>
[2024-03-02T00:05:28.934+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: stock_analyzer.insert_irctc_data_into_db manual__2024-03-02T00:05:22.461486+00:00 [queued]>
[2024-03-02T00:05:28.934+0000] {taskinstance.py:2193} INFO - Starting attempt 1 of 1
[2024-03-02T00:05:28.953+0000] {taskinstance.py:2214} INFO - Executing <Task(PostgresOperator): insert_irctc_data_into_db> on 2024-03-02 00:05:22.461486+00:00
[2024-03-02T00:05:28.962+0000] {standard_task_runner.py:60} INFO - Started process 839 to run task
[2024-03-02T00:05:28.968+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'stock_analyzer', 'insert_irctc_data_into_db', 'manual__2024-03-02T00:05:22.461486+00:00', '--job-id', '145', '--raw', '--subdir', 'DAGS_FOLDER/stock-analyzer.py', '--cfg-path', '/tmp/tmph6y2g7yn']
[2024-03-02T00:05:28.970+0000] {standard_task_runner.py:88} INFO - Job 145: Subtask insert_irctc_data_into_db
[2024-03-02T00:05:29.026+0000] {task_command.py:423} INFO - Running <TaskInstance: stock_analyzer.insert_irctc_data_into_db manual__2024-03-02T00:05:22.461486+00:00 [running]> on host 0a26efe0197d
[2024-03-02T00:05:29.191+0000] {taskinstance.py:2510} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='root' AIRFLOW_CTX_DAG_ID='stock_analyzer' AIRFLOW_CTX_TASK_ID='insert_irctc_data_into_db' AIRFLOW_CTX_EXECUTION_DATE='2024-03-02T00:05:22.461486+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-03-02T00:05:22.461486+00:00'
[2024-03-02T00:05:29.198+0000] {sql.py:276} INFO - Executing: 
            INSERT INTO irctc_stock_data (date, open, high, low, close, volume)
            SELECT date, open::float, high::float, low::float, close::float, volume::float
            FROM '/opt/***/data/irctc_processed.csv'
        
[2024-03-02T00:05:29.218+0000] {base.py:83} INFO - Using connection ID 'psql-aiven' for task execution.
[2024-03-02T00:05:29.251+0000] {base.py:83} INFO - Using connection ID 'psql-aiven' for task execution.
[2024-03-02T00:05:29.436+0000] {sql.py:457} INFO - Running statement: 
            INSERT INTO irctc_stock_data (date, open, high, low, close, volume)
            SELECT date, open::float, high::float, low::float, close::float, volume::float
            FROM '/opt/***/data/irctc_processed.csv'
        , parameters: None
[2024-03-02T00:05:29.483+0000] {taskinstance.py:2728} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/taskinstance.py", line 444, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/taskinstance.py", line 414, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/providers/common/sql/operators/sql.py", line 282, in execute
    output = hook.run(
             ^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/providers/common/sql/hooks/sql.py", line 404, in run
    self._run_command(cur, sql_statement, parameters)
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/providers/common/sql/hooks/sql.py", line 462, in _run_command
    cur.execute(sql_statement)
psycopg2.errors.SyntaxError: syntax error at or near "'/opt/***/data/irctc_processed.csv'"
LINE 4:             FROM '/opt/***/data/irctc_processed.csv'
                         ^

[2024-03-02T00:05:29.494+0000] {taskinstance.py:1149} INFO - Marking task as FAILED. dag_id=stock_analyzer, task_id=insert_irctc_data_into_db, execution_date=20240302T000522, start_date=20240302T000528, end_date=20240302T000529
[2024-03-02T00:05:29.519+0000] {standard_task_runner.py:107} ERROR - Failed to execute job 145 for task insert_irctc_data_into_db (syntax error at or near "'/opt/***/data/irctc_processed.csv'"
LINE 4:             FROM '/opt/***/data/irctc_processed.csv'
                         ^
; 839)
[2024-03-02T00:05:29.566+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 1
[2024-03-02T00:05:29.592+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.11/site-packages/***/models/baseoperator.py:1201: AirflowProviderDeprecationWarning: Call to deprecated class PostgresOperator. (Please use `***.providers.common.sql.operators.sql.SQLExecuteQueryOperator`.Also, you can provide `hook_params={'schema': <database>}`.)
  result = cls.__new__(cls)

[2024-03-02T00:05:29.609+0000] {taskinstance.py:3309} INFO - 0 downstream tasks scheduled from follow-on schedule check
